origin-notifications/README.md

```diff
@@ -5,3 +5,54 @@ Head to https://www.originprotocol.com/developers to learn more about what we're
 # Origin Notifications
 
 See https://github.com/OriginProtocol/origin/issues/806 👊
+
+## Getting Started
+
+### The Easy Way
+
+We recommend [running Origin Box](https://github.com/OriginProtocol/origin/blob/master/DEVELOPMENT.md#using-docker-compose) to simplify the process of configuring and running all of the modules within the Origin Monorepo.
+
+### The Not-As-Easy Way
+
+If you're interested in running just the notifications service (and the event listener) without messaging, attestations, etc, here are some suggestions for making that happen:
+
+
+In origin-notifications...
+
+  - Install and run [PostgreSQL](https://www.postgresql.org/)
+  - Run `npm install web-push -g`
+  - Run `web-push generate-vapid-keys`
+  - Add database url, email address, public and private keys to origin-notifications/.env
+  ```
+  DATABASE_URL=postgresql://localhost/notification
+  VAPID_EMAIL_ADDRESS=XXXXXXXXXX-your-email-address-XXXXXXXXXX
+  VAPID_PRIVATE_KEY=XXXXXXXXXX-your-private-key-XXXXXXXXXX
+  VAPID_PUBLIC_KEY=XXXXXXXXXX-your-public-key-XXXXXXXXXX
+  ```
+  - Run `createdb notification`
+  - Run `npm install -g sequelize-cli`
+  - Run `sequelize db:migrate`
+  - Run `npm run start:development`
+
+In origin-dapp...
+
+  - Add public key and notifications url to origin-dapp/.env
+  ```
+  NOTIFICATIONS_KEY=XXXXXXXXXX-your-public-key-XXXXXXXXXX
+  NOTIFICATIONS_URL=http://localhost:3456/
+  ```
+  - Run `npm start`
+  
+In origin-discovery...
+  
+  - Run `node src/listener/listener.js --continue-file=continue --webhook=http://localhost:3456/events`
+  
```

Comment:
I have at least one undefined variable
![image](https://user-images.githubusercontent.com/17957496/50989515-d1f6b100-14cc-11e9-8399-6688cf0c27f7.png)

I think it is the `DATABASE_URL`.
- Is this the correct format `postgresql://localhost/notification`?
- Is there another variable I need?

Reply:
Ah, I forgot one. Thanks for testing. 😁

Reply:
@Natasha08 try now with 37dd006 🤞